lib/improver/wind_calculations/wind_direction.py

```diff
@@ -427,7 +427,7 @@ def process(self, cube_ens_wdir):
             raise ValueError(msg)
 
         # Force input cube to float32.
```

Comment:
This comment is no longer true. Now the code will only demote float64 to float32, it will not force any data to float32. I think this is the correct change, but the comment needs updating.